spark/hbase-spark/src/test/scala/org/apache/hadoop/hbase/spark/DefaultSourceSuite.scala

```diff
@@ -990,7 +992,8 @@ class DefaultSourceSuite
                      |"DOUBLE_FIELD":{"cf":"c", "col":"double", "type":"double"},
                      |"DATE_FIELD":{"cf":"c", "col":"date", "type":"date"},
                      |"TIMESTAMP_FIELD":{"cf":"c", "col":"timestamp", "type":"timestamp"},
-                     |"STRING_FIELD":{"cf":"c", "col":"string", "type":"string"}
+                     "STRING_FIELD":{"cf":"c", "col":"string", "type":"string"},
```

Comment:
should not this be `|"STRING_FIELD":{"cf":"c", "col":"string", "type":"string"},`

Reply:
Sorry for typo